### PR TITLE
Update README.md and add badges

### PR DESCRIPTION
**Pull request by danielSbastos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +31,9 @@ OAUTH_TOKEN (Github OAuth Token)
 
 Other variables, for example, `COMMIT_ID` and `REPOSITORY_NAME`, are optional because they can be fetched from the local development data. In CIs, like CircleCI, these 5 variables are avaliable without our need to configure them.
 
-#### Dependencies
+### Usage
 
-Download the dependencies with `pip install -r requirements.txt`
+After exporting the environment variables above. Simply execute `$ feito`. In a few moments, the analysis done by Prospector will be shown in your PR
```
Comment (alfakini) [inline README.md]: `feito`

Comment (alfakini) [reply README.md]: Missing dot in the end of sentence.

**Comment by alfakini** [comment]
@danielSbastos let's add some stuff! 

This badge: [code review by feito](https://github.com/magrathealabs/feito)

## Badge

Show the world you're using _Feito_! Add the badge [code review by feito](https://github.com/magrathealabs/feito) to your project.

```md
[![code review by feito](https://img.shields.io/badge/code%20review%20by-feito-blue.svg)](https://github.com/magrathealabs/feito)
```

## License

The package is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).

File: README.md (hunk 1 above)
Comment (diogotrentini) [reply README.md]: > After exporting the environment variables above, execute `feito`.

File: README.md [hunk 2]
```diff
@@ -6,13 +10,15 @@ Automated code review in Python done with Prospector
...
-### 1) Requirements
+### Installation
+
+`$ pip install feito`
 
-An account in GitHub must have its OAuth Token acquired. Follow this [link](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token). When the Scopes section arrives, the `admin:repo_hook` and `repo` checkboxes must be selected. Copy the Token (really, copy it, you won't be able visualize after) and save it somewhere.
+### Requirements for usage
 
-### 2) Setup environment
+In order to have an account to comment as Feito in your PRs, a GitHub OAuth Token must be acquired. Follow this [link](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token). When the Scopes section arrives, the `admin:repo_hook` and `repo` checkboxes must be selected. Copy the Token (really, copy it, you won't be able visualize after) and save it somewhere.
```
Comment (diogotrentini) [inline README.md]: I think "you won't be able to visualize it afterward" would be better.

File: README.md (hunk 1 above)
Comment (danielSbastos) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (danielSbastos) [reply README.md]: Me too.

**Pull request merged**